Ports/.hosted_defs.sh

```diff
@@ -38,6 +40,7 @@ else
     export SERENITY_TOOLCHAIN_BINDIR="${SERENITY_SOURCE_DIR}/Toolchain/Local/${SERENITY_ARCH}/bin"
     export CC="${SERENITY_ARCH}-pc-serenity-gcc"
     export CXX="${SERENITY_ARCH}-pc-serenity-g++"
+    export LD="${SERENITY_TOOLCHAIN_BINDIR}/x86_64-pc-serenity-ld"
```

Comment:
Unlike the other entries, does `${SERENITY_ARCH}-pc-serenity-ld` not work here? It feels unnecessary to forego PATH resolving manually.

Reply:
Right, thank you.